bvm/ballerina-core/src/main/java/org/ballerinalang/util/debugger/dto/VariableDTO.java

```diff
@@ -79,22 +78,10 @@ private String getStringValue(BValue bValue) {
         } else if (bValue instanceof BValueType || bValue instanceof BXML ||
                 bValue.getType().getTag() == TypeTags.JSON) {
             bValueString = bValue.stringValue();
-        } else if (bValue instanceof BNewArray) {
-            BNewArray bArray = (BNewArray) bValue;
-            bValueString = "Array[" + bArray.size() + "] ";
-            bValueString = bValueString + bArray.stringValue();
-        } else if (bValue.getType().getTag() == TypeTags.MAP) {
-            BMap bmap = (BMap) bValue;
-            bValueString = "Map[" + bmap.size() + "] ";
-            bValueString = bValueString + bmap.stringValue();
-        } else if (bValue.getType().getTag() == TypeTags.RECORD) {
-            bValueString = "Record " + bValue.getType().getName() + " ";
-            bValueString = bValueString + bValue.stringValue();
         } else if (bValue.getType().getTag() == TypeTags.OBJECT) {
-            bValueString = "Object " + bValue.getType().getName() + " ";
-            bValueString = bValueString + ((BMap) bValue).absoluteStringValue();
+            bValueString = ((BMap) bValue).absoluteStringValue();
```

Comment:
I checked the output of "absoluteStringValue" it isnt a valid json value. Keys needs to be surrounded by quotes. 
```js
{
	config: {
		host: "0.0.0.0",
		keepAlive: "AUTO",
		secureSocket: null,
		httpVersion: "1.1",
		requestLimits: null,
		filters: [],
		timeoutMillis: 120000,
		maxPipelinedRequests: 10,
		authProviders: null,
		positiveAuthzCache: {
			enabled: true,
			capacity: 100,
			expiryTimeMillis: 5000,
			evictionFactor: 1.0
		},
		negativeAuthzCache: {
			enabled: true,
			capacity: 100,
			expiryTimeMillis: 5000,
			evictionFactor: 1.0
		}
	},
	filterContext: {
		serviceRef: {},
		serviceName: "myService1$$service$0",
		resourceName: "foo",
		attributes: {}
	},
	remoteAddress: {
		host: "localhost",
		port: 44740
	},
	localAddress: {
		host: "localhost",
		port: 9090
	},
	protocol: "http"
}
```